go/vt/vtgate/executor_select_test.go

```diff
@@ -4152,6 +4151,70 @@ func TestSelectView(t *testing.T) {
 	utils.MustMatch(t, wantQueries, sbc.Queries)
 }
 
+func TestWarmingReads(t *testing.T) {
+	ctx := context.Background()
+	executor, primary, replica := createExecutorEnvWithPrimaryReplicaConn(t, ctx, 100)
+
+	executor.normalize = true
+	session := NewSafeSession(&vtgatepb.Session{TargetString: KsTestUnsharded})
+
+	_, err := executor.Execute(ctx, nil, "TestSelect", session, "select age, city from user", map[string]*querypb.BindVariable{})
```

Comment:
The method we pass in should be the name of the test that is caling the Execute, so it should be `TestWarmingReads`.

```suggestion
	_, err := executor.Execute(ctx, nil, "TestWarmingReads", session, "select age, city from user", map[string]*querypb.BindVariable{})
```